sufia-models/lib/generators/sufia/models/templates/config/sufia.rb

```diff
@@ -105,8 +105,11 @@
   # If you have ffmpeg installed and want to transcode audio and video uncomment this line
   # config.enable_ffmpeg = true
 
-  # Specify the Fedora pid prefix:
-  # config.id_namespace = "sufia"
+  # To disable minting of NOIDs for files and collections in favor of Fedora UUIDs
```

Comment:
Change text to `Sufia uses NOIDs for files and collections instead of Fedora UUIDs`